workers/server/worker.go

```diff
@@ -102,6 +102,36 @@ func (w *WorkServer) GetWork(workInst *pb.WorkerInstance, serv pb.Worker_GetWork
 	return nil
 }
 
+// GetGitRepo checks out the code for a git repository.
+func (w *WorkServer) GetGitRepo(ctx context.Context, in *pb.PipelineID) (*pb.GitRepo, error) {
+	repo := &pb.GitRepo{}
+	// Check if worker is registered
```

Comment:
Would love to see some blanket lines between code blocks to make it more readable 😄

Reply:
Don't worry dude, I will make this pretty. :D